Lib/test/test_pdb.py

```diff
@@ -258,6 +258,8 @@ def test_pdb_breakpoint_commands():
     ...     'clear 3',
     ...     'break',
     ...     'condition 1',
+    ...     'commands 1',
+    ...     'EOF',       # Simulate Ctrl-D/Ctrl-Z from user, should end input
```

Comment:
```suggestion
    ...     'EOF',       # Received from cmd on Ctrl-D/Ctrl-Z from user, should end input
```

Reply:
Well technically that's not correct. The explicit `EOF` command here triggers the same callback as the user inputs `Ctrl-D` or `Ctrl-Z`. Both of them triggers `do_EOF`. So it's not accurate to say that `'EOF'` is received from cmd on Ctrl-D from user, because it's not - it's the user typing `'EOF'` then `enter`. It just happens to trigger the same callback and that's how we simulate user using `Ctrl-D`.